DigitalSignageUAP/SlideshowPage.xaml.cs

```diff
@@ -307,8 +307,20 @@ void PlayAudio()
                             DisplayObject DO = new DisplayObject();
                             string filename = fileElement.Attribute("path").Value.Substring(fileElement.Attribute("path").Value.LastIndexOf('/') + 1);
```

Comment:
why don't we use the absolute path here? if the file is in a subfolder in the storage file location, this code will pick up the wrong file...